src/authorization.ts

```diff
@@ -13,13 +13,13 @@ export interface AuthorizationRequest {
     resourceType?: string;
     id?: string;
     vid?: string;
-    export?: ExportAuth;
+    bulkDataAuth?: BulkDataAuth;
 }
 
-export interface ExportAuth {
-    operation: 'initiate-export' | 'get-status' | 'cancel-export';
-    jobRequesterUserId?: string;
-    type?: ExportType;
+export interface BulkDataAuth {
+    operation: 'initiate-export' | 'initiate-import' | 'get-status' | 'cancel';

```

Comment:
We may need to know what type of cancel? right like do we cancel an import or export or do you think the jobid is unique enough?

Reply:
If we use the same DDB table for export and import then just `jobId` should be unique enough, but if we use two different tables then we would need the operations to be more specific. I'll add types for both import and export.